articles/hardware/breakout/analog-io.md

```diff
@@ -0,0 +1,52 @@
+---
+uid: breakout_analog-io
+title: Breakout Board Analog IO
+hardware: true
+device: analog io
+videoCaption: This visualizes analog IO data using Bonsai visualizers. The "BreakoutAnalogOutput" window shows the values that are being used by the <code>BreakoutAnalogOutput</code> operator to update the analog IO port. However, only analog channel 0 is actually updated because it's the only channel configured as output by the <code>BreakoutBoard</code> operator. The "AnalogData" window shows the voltages at the various pins of the analog IO port. They are being read by the the ADCs on the breakout board and produced in Bonsai by the <code>BreakoutAnalogInput</code> operator. This confirms that only analog channel 0 is outputting the sawtooth waveform.
+---
+
+The following excerpt from the [Breakout Board example workflow](xref:breakout) demonstrates analog IO functionality by computing a ~1Hz sawtooth pattern, outputting it to the analog IO port, and reading it back from the analog IO port. It also saves analog inputs data. Though this excerpt comprises of two smaller discrete graphs, they are combined on this page because the easiest way to demonstrate the analog input is to read back analog output data.
```

Comment:
The paragraph starting with "The following except" shows up in many files. Is it worth adding this to the template?

Reply:
I thought about that but the rest of that paragraph is mostly custom-written and not reused too much, so I figured it would be better not to mix template and custom-written content too much.